charts/nri-prometheus/templates/configmap.yaml

```diff
@@ -9,10 +9,17 @@ data:
   config.yaml: |
     cluster_name: {{ include "nri-prometheus.cluster" . }}
 {{- if .Values.config }}
-{{- if (include "nri-prometheus.lowDataMode" .) }}
-{{ $lowDataDefault := .Files.Get "static/lowdatamodedefaults.yaml" | fromYaml -}}
-    {{- mergeOverwrite .Values.config $lowDataDefault | toYaml | indent 4 -}}
-{{- else }}
-    {{ toYaml .Values.config | indent 4 | trim }}
+    {{ omit .Values.config "transformations" | toYaml | indent 4 | trim -}}
```

Comment:
I would `trim` before `indent` and use `{{-` with `nindent` so it will clean all the YAML and do the indentantion properly an consistently.